frontend/package.json

```diff
@@ -17,9 +17,9 @@
     "build-test-pa11y": "./development-configurations/build-test-pa11y.sh",
     "snyk-test": "snyk test",
     "start": "ng serve",
-    "test:ci": "ng test --karma-config ./development-configurations/karma.conf --code-coverage --watch=false --source-map=false --progress=false",
-    "test": "ng test --karma-config ./development-configurations/karma.conf",
-    "test:coverage": "ng test --karma-config ./development-configurations/karma.conf --watch=false --source-map --code-coverage",
+    "test:ci": "NODE_ENV=test ng test --karma-config ./development-configurations/karma.conf --code-coverage --watch=false --source-map=false --progress=false",
+    "test": "NODE_ENV=test ng test --karma-config ./development-configurations/karma.conf",
+    "test:coverage": "NODE_ENV=test ng test --karma-config ./development-configurations/karma.conf --watch=false --source-map --code-coverage",
```

Comment:
do these need to be `export NODE_ENV=test`?

Reply:
No, since this is a single command, specifying the env variables as part of the command will set the environment variables for the shell that runs `ng`. `export` would be needed if we were running multiple commands e.g. `&&` or `command; command`